auth-web/src/store/modules/org.ts

```diff
@@ -284,6 +284,26 @@ export default class OrgModule extends VuexModule {
     this.resetAccountTypeOnSetupAccount = resetAccountTypeOnSetupAccount
   }
 
+  @Mutation
+  public setCurrentOrganizationIsBusinessAccount (isBusinessAccount: boolean) {
```

Comment:
all these mutations can be combined to one? since pushing into this.currentOrganization?